src/app/components/currency/currency.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react';
+import Currency, { CurrencyProps } from './currency';
+
+describe(Currency.name, () => {
+  test('should format the given amount in INR currency format', () => {
+    const props: CurrencyProps = {
+      amount: 12345
+    };
+
+    render(<Currency {...props}></Currency>);
+
+    screen.getByText('₹12,345.00');
+  });
+
+  test('should set a span with empty text, if amount is not sent', () => {
+    const props: CurrencyProps = {
+    };
+
+    const { container } = render(<Currency {...props}></Currency>);
+
+    const emptySpan = container.querySelector('span');
+    expect(emptySpan?.textContent).toBe('');
```

Comment:
can emptySpan be null/undefined?

Reply:
I see what you mean here, probably testing that the span is available and then a check to see if the text is empty would add a lot of meaning ryt